Changelog.md

```diff
@@ -38,6 +36,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
  * Add `MetricsExporterResources` and `MySQLOperatorSidecarResrouces` in `.Spec.PodSpec` to allow
    the user specifying resources for thos sidecars containers.
  * Add command line flag to configure number of workers for orchestrator controller.
+ * Add `backupAffinity`, `backupNodeSelector`, `backupPriorityClassName`, `backupTolerations`
+   to `.Spec.PodSpec` to allow specifying custom scheduling constraints for backup jobs.
```

Comment:
Please add those into the `Unreleased` section above because those changes will be included into the next release.

Reply:
@AMecea: done!